lib/marriage_abroad_outcome_flattener.rb

```diff
@@ -0,0 +1,90 @@
+class MarriageAbroadOutcomeFlattener
+  COUNTRIES_DIR = "lib/smart_answer_flows/marriage-abroad/outcomes/countries".freeze
+
+  FEE_TABLE_PARTIAL = <<~FEE_TABLE.freeze
+    <%= render partial: 'consular_fees_table_items.govspeak.erb',
+        collection: calculator.services,
+        as: :service,
+        locals: { calculator: calculator } %>
+  FEE_TABLE
+
+  HOW_TO_PAY_PARTIAL = <<~HOW_TO_PAY.freeze
+    <%= render partial: 'how_to_pay.govspeak.erb', locals: {calculator: calculator} %>
+  HOW_TO_PAY
+
+  TITLE_PARTIAL_CONTENT = <<~TITLE.freeze
+    <% if calculator.partner_is_same_sex? %>
+      Civil partnership in COUNTRY
+    <% else %>
+      Marriage in COUNTRY
+    <% end %>
+  TITLE
+
+  PAYMENT_RE = /^\^?You can( only)? pay by.*?\n.*?^$/mi
+  SERVICE_FEE_RE = /^Service \| Fee\n.*?^$/mi
+
+  attr_reader :country, :logger
+
+  def initialize(country, logger = STDOUT)
+    @country = country
+    @logger = logger
```

Comment:
Are you anticipating any other loggers than STDOUT? I notice the Rails one has `puts` as a private method and that seems to be only method we call on the logger.